ChangeLog.d/00README.md

```diff
@@ -86,6 +86,6 @@ See [existing entries](../ChangeLog) for examples.
 
 ## How `ChangeLog` is updated
 
-Run [`../scripts/assemble_changelog.py`](../scripts/assemble_changelog.py)
+Run [`../framework/scripts/assemble_changelog.py`](../scripts/assemble_changelog.py)
```

Comment:
```suggestion
Run [`../framework/scripts/assemble_changelog.py`](../framework/scripts/assemble_changelog.py)
```

Reply:
Ops, sorry, I definitely overlooked this.